domain/lib/src/usecases/download_file/download_file_exception.dart

```diff
@@ -29,18 +29,34 @@
 //  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
 //  the Additional Terms applicable to LinShare software.
 
-class DeviceNotSupportedException implements Exception {
-  String notSupported = 'This device is not supported, please try on Android or iOS';
-}
+import 'package:equatable/equatable.dart';
 
-class DownloadFileException implements Exception {
+abstract class DownloadFileException extends Equatable implements Exception {
```

Comment:
why not `with Equaltable`

Reply:
I don't know to be honest I'm not familiar with mixin so that's why I used extends